custom/icds/messaging/indicators.py

```diff
@@ -244,6 +246,12 @@ def __init__(self, domain, user):
         ).aggregate(value=Max("last_form_submission_date"))["value"]
 
     def get_messages(self, language_code=None):
+        # default 24 hours
+        ACCEPTABLE_WAREHOUSE_LAG_IN_MINUTES = getattr(settings, 'ACCEPTABLE_WAREHOUSE_LAG_IN_MINUTES', 60 * 24)
+        warehouse_lag = (datetime.utcnow() - get_warehouse_latest_modified_date()).total_seconds() / 60
+        if warehouse_lag > ACCEPTABLE_WAREHOUSE_LAG_IN_MINUTES:
+            return []
```

Comment:
It will be nice to log when this skips the message, but the way it's setup, we would log a message for each user.

Reply:
There might be some way to poll this info and send to datadog, I can do that in a separate PR latter.